.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish Release
+
+permissions:
+  id-token: write
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    name: Perform Publishing
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
Just for my understanding. The `release` trigger will make sure that the tagged commit from the release will be checked out here? (Probably via `GITHUB_SHA` env? https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release)